src/Apps/Order/Routes/__tests__/Offer.test.tsx

```diff
@@ -54,13 +54,13 @@ describe("Offer InitialMutation", () => {
     expect(transactionSummary.text()).toContain("Your offer")
 
     input.props().onChange({ currentTarget: { value: "1" } } as any)
-    expect(transactionSummary.update().text()).toContain("Your offer$1")
+    expect(transactionSummary.text()).toContain("Your offer$1")
 
     input.props().onChange({ currentTarget: { value: "1.23" } } as any)
-    expect(transactionSummary.update().text()).toContain("Your offer$1")
+    expect(transactionSummary.text()).toContain("Your offer$1")
 
     input.props().onChange({ currentTarget: { value: "1023.23" } } as any)
-    expect(transactionSummary.update().text()).toContain("Your offer$1,023")
```

Comment:
oooh so this was a bug represented by the other bug -- i had no idea how this worked before and was so confused.

Reply:
Somewhat, yeah, but there’s tons of tests where (apparently) we already needed to update the tree after state changes, so these felt natural. But yep, the need went away after the fix 👍